class6/DistinctElements.java

```diff
@@ -5,16 +5,52 @@
  *  input = [3,4,6,2,1,6,8] - False
  *  input = [2,3,1,7,6] - True
  */
+import java.util.*;
 public class DistinctElements {
     private static boolean isDistinct(int[] nums){
+    	if(nums == null)
+    		return true;
+    	int arrayLength = nums.length;
+    	if (arrayLength == 0){
+    		return true;
+       	}
+    	boolean isDistinct = true;
+    	HashMap<Integer, Integer> count = new HashMap<Integer, Integer>();
+    	for(int i=0 ; i < nums.length; i++){    		
+    				
+    			int index = nums[i];
+    			if(count.get(index) == null){
+    				count.put(index,1);
+    			}
+    			else{
+    				
+    				isDistinct = false;
```

Comment:
Good try.. solution in O(n) which is good..
you can return here, instead of break?  and return true outside the loop?